packages/dr.rb

```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class dr < Package
+  description 'A simple ed-like devRant client'
+  homepage 'https://github.com/Ewpratten/dr'
+  version '1.0-cros'
+  source_url 'https://codeload.github.com/Ewpratten/dr/tar.gz/1.0-chr'
+  source_sha256 '3e7b33bec353b23132db365e5bacd175361d1e9039544e5e64b8df8c7e828006'
+
+  depends_on 'python3'
+
+  def self.build
+    system "cd build/"
+    system 'sh ./build-cros.sh'
```

Comment:
Commands do not work as you might expect when using system.  The `sh ./build-cros.sh` line is actually still in the root directory, not in build.  A better way to approach this would be:
```
- system "cd build/"
- system 'sh ./build-cros.sh'
+ Dir.chdir 'build' do
+   system 'sh ../build-cros.sh'
+ end
```

Reply:
The 'cd' command doesn't work in crew. Look at dpkg for an example.